years.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+if (grep -r "Copyright[[:space:]]\+(c)[[:space:]]\+2009-.*rultor.com" \
+  --exclude-dir "target" --exclude-dir ".git" --exclude "years.sh" . \
+  | grep -v 2009-`date +%Y`) then
+    echo "Files above have wrong years in copyrights"
+    exit 1
+fi
+
+if (grep -r -L "Copyright[[:space:]]\+(c)[[:space:]]\+2009-.*rultor.com" \
+ --include=*.{java,xml,vm,groovy,txt,fml,properties} --exclude-dir "target" . \
```

Comment:
@mkordas technically, offcource, possible. First I did so, but didn't like the decision because it was full of custom exclusions. Than I proposed another desicion described [here](https://github.com/yegor256/rultor/issues/981#issuecomment-189681250) and got approve. Offcource, I won't be happy to change it back - there is no profit for me and I don't think, I did something wrong.